src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/add-user-service/add-user-service-checklist)
Custom AuthenticationException **shouldn’t** be RuntimeException.